content/en/observability_pipelines/monitoring/_index.md

```diff
@@ -0,0 +1,73 @@
+  tag: "Documentation"
+  text: "Configure a metric monitor"
+- link: "/observability_pipelines/monitoring/metrics/"
+  tag: "Documentation"
+  text: "Observability Pipelines metrics"
+---
+
+## Overview
+
+A pipeline consists of components that collect, process, and route your observability data. You can track the status of your pipelines and components in the following ways:
+
+- View health graphs of your [pipelines](#view-the-status-of-your-pipelines), [Workers](#view-the-status-of-your-workers), and [components](#view-the-status-of-your-pipeline-components) (sources, processors, and destinations).
+- Enable [out-of-the-box monitors](#out-of-the-box-monitors) that alert you if:
+    - An Observability Pipelines Worker has high CPU or memory usage, or is dropping data.
+    - A component is emitting errors.
+    - A defined quota has been reached.
+- Create your own dashboards, notebooks, and monitors with the available [Observability Pipelines metrics][5].
+
+## View the status of your pipelines
+
+1. Navigate to [Observability Pipelines][1] to see how many bytes your pipelines are receiving and sending out.
+1. Select a pipeline.
+1. Click the **Health** tab to see details about the pipeline and its components. You can view graphs of:
+    - How much each component is being used, and the total number of events that the component receives and sends out.
+    - The number of requests made to destinations, and the number of errors encountered by those requests.
+    - How many events are intentionally and unintentionally discarded.
+    - Any changes in the number of requests and errors for each component over the previous week.
+
+You can export a health graph to a dashboard, notebook, or monitor. The exported graph shows you that the metric is grouped by the specific pipeline and component tags.
+
+## View the status of your Workers
+
+To view graphs of resource usage and data sent through Observability Pipelines Workers:
+
+1. Navigate to [Observability Pipelines][1].
+1. Select a pipeline.
+1. Click the **Workers** tab to see the Workers' memory and CPU utilization, traffic stats, and any errors.
+
+## View the status of your pipeline components
+
+To view metrics for a source, process, or destination:
+
+1. Navigate to [Observability Pipelines][1].
+1. Select a pipeline.
+1. Click the cog next to the source's, processor's, or destination's name, then select **View details**. Datadog displays health graphs for the component you selected.
+1. If you want to export a graph to an [incident][2], [dashboard][3], or [notebook][4], click the export icon on the graph. The exported graph shows that the metric is grouped by the specific pipeline and component tags.
+
+## Out-of-the-box monitors
+
+To see available out-of-the-box monitors:
+
+1. Navigate to [Observability Pipelines][1].
+1. Click **Enable monitors** in the **Monitors** column for your pipeline.
+1. Click **Enable** to set up a monitor for that use case.<br>
```

Comment:
Maybe instead of "that use case" it could be something like "one of the suggested use cases"?

Also I see **Start** and not **Enable**, are we seeing different things?

Reply:
Ah, I see **Start** too. Updated that and also used your suggestion regarding "that use case".